test/quantization/test_quantize_script.py

```diff
@@ -1113,20 +1087,27 @@ class TestQuantizeScriptPTSQOps(QuantizationTestCase):
     """ Test graph mode post training static quantization works
     for individual ops end to end.
     """
-    def _test_op_impl(self, module, data, quantized_op, debug=False):
+    def _test_op_impl(self, module, data, quantized_op, check=True):
```

Comment:
Suggest refactoring this into two parts:
1. Script/Trace the model and create the outputs dict. 
2. Check for specific ops in the output dict